lib/docusign_rest/client.rb

```diff
@@ -426,6 +426,22 @@ def get_tabs(tabs, options, index)
       tab_array
     end
 
+    # Allows adding extra attributes to the file array
+    def create_file_ios_with_params(files)
+      ios = []
```

Comment:
And until we do go that route, I'd like to keep things as consistent as possible with the original. If we need to make more changes down the road, we will start refactoring.